src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj

```diff
@@ -46,8 +46,8 @@
     <Reference Include="Microsoft.Azure.Management.Compute">
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Compute.21.0.0\lib\net452\Microsoft.Azure.Management.Compute.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.3.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=20.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
tqalk to @MikhailTryakhov  about the version, I think the latest version he is releasing is 20.0.0. All the PRs going into the September branch need to use that version

Reply:
@praries880 - Yes that is correct, I will update this PR when the version is available.

Reply:
.NET SDK is merged and (with a bit of magic) finally published https://www.nuget.org/packages/Microsoft.Azure.Management.Network/19.4.0-preview 

So now all contributors must change SDK version from local version (localFeed folder) to 19.4.0 and remove nuget packages from localFeed folder.